core/events/services/classic/models.py

```diff
@@ -67,6 +67,10 @@ class Submission(Base):    # type: ignore
     WITHDRAWN_FORMAT = 'withdrawn'
```

Comment:
There may be a typo in the value for `WITHDRAWAL` (should be `wdr` I think)

Reply:
https://github.com/cul-it/arxiv-submission-core/issues/24